src/platform/Linux/CHIPBluezHelper.h

```diff
@@ -205,6 +205,58 @@ CHIP_ERROR StopDiscovery(BluezEndpoint * apEndpoint);
 
 CHIP_ERROR ConnectDevice(BluezDevice1 * apDevice);
 
+/// Iterates over available BlueZ adapters
+///
+/// Usage example:
+///
+///  AdapterIterator iterator;
+///  while (iteerator.Next()) {
+///      std::cout << iterator.GetAddress() << std::endl;
+///  }
+class AdapterIterator
+{
+public:
+    ~AdapterIterator();
+
+    /// Moves to the next DBUS interface.
+    ///
+    /// MUST be called at least once
+    bool Next();
+
+    // information about the current value
```

Comment:
This is only safe to call... when?  Ideally it would be safe to call if `Next()` just returned `true`, right?

Reply:
Updated comment.